### ISSUE TITLE
iframes not clipped by border radius

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#9983]
Affects browserhtml

`border-radius` on a iframe and its parents doesn't clip the content of the iframe.

Happens both with and without webrender.

``` html
<style>
  div {
    display: inline-block;
    border-radius: 0 0 50px 50px;
    border: 5px solid red;
    overflow: hidden;
  }
  iframe {
    border: 5px solid green;
    display: block;
    height: 100px;
    width: 100px;
    border-radius: 50px 50px 0 0;
  }
</style>
<div>
  <iframe src="data:text/html,<style>body{background:orange}</style>"></iframe>
</div>
```

<img width="566" alt="screen shot 2016-03-14 at 06 17 13" src="https://cloud.githubusercontent.com/assets/373579/13735664/9d9202e6-e9ac-11e5-9156-d8bb60c52a33.png">

**Comment by pcwalton** [comment]
We actually don't support clipping iframes *at all*. The appropriate clip region isn't being communicated to WebRender, and WebRender isn't using it. Working on a fix.

**Comment by mrobinson** [comment]
Assigning myself here as well, because I've been working on this as well and should be able to have a fix rather quickly.

**Comment by mrobinson** [comment]
This should be fixed by #17771.
